src/content/1.7/development/webservice/tutorials/3-prestashop-webservice-lib/1-install-library.md

```diff
@@ -0,0 +1,116 @@
+---
+title: 1 - Accessing the Webservice
+weight: 1
+---
+
+# PrestaShop Webservice lib
+
+Now that your webservice is configured and accessible you might want to use it. To help you perform request on your webservice we provide a [PHP library for PrestaShop Webservices](https://github.com/PrestaShop/PrestaShop-webservice-lib).
+
+## Install the lib
+
+### Pre-requisites
+
+* You need a server with `mod_rewrite` enabled
+* You need the `curl` extension enabled in PHP
+
+### Installing with Composer
+
+If you are starting a new project you can init your composer project along with the dependency:
+
+```bash
+composer init --require="Prestashop/Prestashop-webservice-lib:dev-master" -n  
+composer install
+```
+
+Or if you already have an existing project simply add the dependency:
+
+```bash
+composer require "Prestashop/Prestashop-webservice-lib"  
+```
+
+The library is not PSR compliant and has no namespace therefore you need to update your `composer.json` file to include the class in your `autoload`:
+
+```json
+{
+    "require": {
+        "Prestashop/Prestashop-webservice-lib": "dev-master"
+    },
+    "autoload": {
+        "files": [
+            "vendor/prestaShop/prestaShop-webservice-lib/PSWebServiceLibrary.php"
+        ]
+    }
+}
+```
+
+And regenerate your `autoload`:
+
+```bash
+composer dump-autoload
+```
+
+Then you can use the library via autoloading:
```

Comment:
```suggestion
Then you can use the library with Composer's autoloading:
```

Alternatively, the file could be imported

Reply:
`Alternatively, the file could be imported` you mean like in the manual install?

Reply:
Exactly.

Reply:
Ok, so it's already describe in the manual section